test/integration/server0_p_10_p_1_test.exs

```diff
@@ -11,40 +11,54 @@ defmodule KafkaEx.Server0P10P1.Test do
     # note this checks against the version of broker we're running in test
     # api_key, max_version, min_version
     api_versions_kafka_0_11_0_1 = [
-      [0, 3, 0],
```

Comment:
Are we testing against a newer version of Kafka now?  If so, can we update the README (IIRC it states what version we test against) and the variable names here (`kafka_0_11_0_1` etc)?